docs/commands/gnexti.md

```diff
@@ -0,0 +1,11 @@
+## Command `gnexti`
+
+The `gnexti` command allows you to easily step over instructions execution.
```

Comment:
Please add what this command actually does (continuing to temporary breakpoint at next instruction).

Reply:
done!